apps/core/tailwind.config.ts

```diff
@@ -83,6 +83,9 @@ export default {
 			scale: {
 				'101': '1.01',
 			},
+			backdropBlur: {
+				5: '20px',
```

Comment:
can you work with @mystie711 to define some proper values for this? e.g. `s` `md` `lg` `xl` etc.

Reply:
backdrop-blur already has these values, I am unable to fit `20px` in otherwise betweeen `lg` (16px) and `xl` (24px)

Reply:
in that case can we override `lg` to be `20px`

Reply:
I will keep the name as `backdrop-blur-networkSwitcher` since `lg` is already used elsewhere and I don't want to override default value from tailwind.

Reply:
ideally these values would be consistent in the designs across our apps, eventually we're going to have a config file full of one-offs!

Reply:
Yea, let me sync with Abhi tomorrow to see if we can use the existing `lg` value.

Reply:
let's use the following variables for blur,
small 8, medium 12, regular 20, large 24, x-large 32

i will capture and update it so in Figma designs as well going forward.

Reply:
Updated the config to these values